app/broadcast_message/rest.py

```diff
@@ -101,7 +106,7 @@ def create_broadcast_message(service_id):
         template_id=template.id,
         template_version=template.version,
         personalisation=data.get('personalisation', {}),
-        areas=data.get('areas', []),
+        areas=data.get('areas', {}),
```

Comment:
Does this mean the interface for creating a broadcast is different to updating one.

So to create one I would post

```
{"areas": {"areas": [things here], "simple_polygons": [things here]}}
```

but to update it I would post
```
{"areas": [things here], "simple_polygons": [things here]}
```

If that is so then I don't think I like having that difference. Would you agree?

Simarly, if we were to create a broadcast, we could create it with just `areas` at the moment but nothing would stop us not giving `simple_polygons`. 

Essentially, I want the type of data and the validation rules to be consistent for creating and updating please.

Feel free to let me know if I've got this wrong or you don't think this is worth it. Sorry I didn't spot this earlier and there has been some back and forth from me stopping this going out so far.

Reply:
Ahh, I know why the disparity: when creating broadcast message, we are not specifying areas at all, we are not sending them over from admin.

But you are right I should update this statement in rest to be consistent and future-proof 👍